### ISSUE TITLE
rent statistics

### ISSUE THREAD

**Issue by JSmithOner** [RealT-Community/dashboard-v2#29]
I am at this moment working on a statitics canvas based on previous transaction.
Do you approve this idea?

Basically this should show the progression of rents in time and aquisition time of tokens.

**Comment by jycssu-com** [comment]
I don't think this is currently possible to implement. The API only returns information on current rents and not history. It would be necessary either to have the possibility of fetching the evolution of rents by property, or to have an API which is based on the rent CSV files provided by Realt. It seems to me that the second option is under development on the API side

**Comment by JSmithOner** [comment]
@jycssu-com you are are right and I just realized it, the graph would be innacurate. I know cryptalloc keeps track of changes. Any API for that?

**Comment by NandyBa** [comment]
Community website https://www.cryptalloc.com/realtlab/ track rent change

**Comment by JSmithOner** [comment]
@NandyBa Yes I know but is there any API that could be used for the community dashboard?Thanks in advanve

**Comment by ehpst** [comment]
Would this be of interest in the meantime https://ehpst.duckdns.org/realt_rent_tracker/api/rent_holder/0x... ? Long term I think we should have a community database with all rent data taken from CSV files.

**Comment by Sigri44** [comment]
> Would this be of interest in the meantime https://ehpst.duckdns.org/realt_rent_tracker/api/rent_holder/0x... ? Long term I think we should have a community database with all rent data taken from CSV files.

Already exist

**Comment by ehpst** [comment]
Where is it then? Are we displaying rent data anywhere?

**Comment by JSmithOner** [comment]
@ehpst I've stopped working on the dashboard for the moment.
